flexmeasures/data/migrations/versions/650b085c0ad3_consolidate_data_source_after_storage_.py

```diff
@@ -0,0 +1,27 @@
+"""Consolidate data source after storage schedulers merged
+
+Revision ID: 650b085c0ad3
+Revises: 30f7b63069e1
+Create Date: 2022-11-16 07:07:44.281943
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "650b085c0ad3"
+down_revision = "30f7b63069e1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        "Update data_source set version='1', model='StorageScheduler' where name = 'Seita' and type='scheduling script';"
+    )
+
+
+def downgrade():
+    op.execute(
```

Comment:
I'd suggest to Just add a little confirmation dialogue here, similar to what we did in `7f8b8920355f_create_annotation_table.py`, to warn for the (potential) loss of data:

```
import click

    click.confirm(
        "This downgrade drops the version and model information from Seita scheduler data sources. Continue?",
        abort=True,
    )
```

Technically, this downgrade only results in loss of data in case there are multiple models or versions of schedulers from Seita in the database, and we could check for that. But I'd understand if you want to move on.

Reply:
How about this instead:

`Update data_source set version=null, model=null where name = 'Seita' and type='scheduling script' and version='1' and model='StorageScheduler';`

Reply:
Right. That works, too.